### DISCUSSION TITLE
What status means what?

### DISCUSSION THREAD

**Discussion by fritzrehde** [transmission/transmission#5343] [Q&A] [answered]
If I run `transmission-remote --list`, I can see the status code of each torrent. However, I do not know which status code means what. Where can I find documentation that explains what each status code means?

**Comment by ile6695** [comment, marked as answer]
You can find it from [docs/rpc-spec](https://github.com/transmission/transmission/blob/main/docs/rpc-spec.md)

It has a lot of stuff, so here's the status codes section.
Value | Meaning
-- | --
0 | Torrent is stopped
1 | Torrent is queued to verify local data
2 | Torrent is verifying local data
3 | Torrent is queued to download
4 | Torrent is downloading
5 | Torrent is queued to seed
6 | Torrent is seeding

**Reply by fritzrehde** [reply]
Thanks!
